specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1138,7 +1138,7 @@ Connector architecture.
 The JDBC 4.2 specification is available at
 _https://jcp.org/en/jsr/detail?id=221_ .
 
-===== [[a2553]]Jakarta™ API for XML Web Services (JAX-WS) Requirements
+===== [[a2553]]Jakarta XML Web Services (JAX-WS) 2.3 Requirements
```

Comment:
I would not add "more information" like the 2.3 version.  I know this is being picky, but if we didn't specify a version previously, then don't add the version.